{"name": "Нельзя авторизоваться без поля name в теле запроса", "status": "passed", "description": "Для ревьюера: здесь баг, в документации сказано, что если не передать одно из полей,вернется ошибика 401, к полю name это тоже относится.", "start": 1719570396813, "stop": 1719570397653, "uuid": "ad559ba2-ad40-41c7-a619-0563042f2761", "historyId": "60022ad857f2cf01941b0dde4a88f9a4", "testCaseId": "60022ad857f2cf01941b0dde4a88f9a4", "fullName": "tests.test_login_user.TestLoginUser#test_login_user_without_field_name_code_401", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "DESKTOP-USF6L0H"}, {"name": "thread", "value": "15232-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_user"}]}